<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 16px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 14px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<div class="rtl-content">

# 🌍 بررسی باندهای ماهواره Sentinel-2

در این بخش، توضیحاتی درباره هر باند داده‌های ماهواره Sentinel-2 ارائه می‌شود. این باندها برای تحلیل‌های مختلفی مانند پوشش گیاهی، منابع آب، وضعیت خاک و ... به کار می‌روند.

</div>


<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 14px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<div class=rtl-content>

## 📊 جدول باندهای Sentinel-2 و کاربرد آن‌ها

| باند | نام | طول موج (نانومتر) | کاربرد اصلی |
|------|-----|--------------------|--------------|
| **B1** | Coastal aerosol | 443 | تصحیح اتمسفری، ذرات معلق در آب و هوا |
| **B2** | Blue | 490 | بررسی ویژگی‌های آب کم‌عمق، تصحیح رنگ |
| **B3** | Green | 560 | **تشخیص آب و پوشش گیاهی**، ورودی در NDWI و MNDWI |
| **B4** | Red | 665 | **تشخیص پوشش گیاهی**، استفاده در NDVI |
| **B5** | Red Edge 1 | 705 | ارزیابی سلامت گیاهان، سنجش کلروفیل |
| **B6** | Red Edge 2 | 740 | تحلیل تنش زیستی گیاهان |
| **B7** | Red Edge 3 | 783 | بررسی وضعیت زیستی پوشش گیاهی |
| **B8** | NIR | 842 | **پوشش گیاهی و آب**، کاربرد در NDVI و NDWI |
| **B8A** | Narrow NIR | 865 | پایش دقیق محصولات کشاورزی |
| **B9** | Water vapor | 945 | شناسایی بخار آب در جو، تصحیح‌های جوی |
| **B10** | SWIR – Cirrus | 1375 | حذف ابرهای نازک (Cirrus) از تصاویر |
| **B11** | SWIR 1 | 1610 | **رطوبت خاک، مناطق خشک و شهری**، استفاده در MNDWI |
| **B12** | SWIR 2 | 2190 | بررسی آتش‌سوزی‌ها، خشکی و رطوبت سطحی |

</div>


<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 14px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<div class=rtl-content>

## 🧠 نکات کلیدی

- **پوشش گیاهی:** استفاده از باندهای **Red (B4)** و **NIR (B8)** برای NDVI.

- **تشخیص آب:** ترکیب **Green (B3)** با **NIR (B8)** یا **SWIR (B11/B12)** برای NDWI و MNDWI.

- **مناطق شهری یا خشک:** باندهای **SWIR** بازتاب ویژه‌ای دارند و در تحلیل MNDWI نقش مهمی دارند.
</div>

---

<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 14px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<div class=rtl-content>

### 🛰️ تحلیل شاخص‌های NDVI، NDWI و MNDWI با استفاده از Sentinel-2

#### الف. فرمول NDWI و باندهای مربوطه

شاخص NDWI (Normalized Difference Water Index) برای شناسایی آب‌های سطحی طراحی شده است. فرمول آن به شکل زیر است:

$$
NDWI = \frac{Green - NIR}{Green + NIR}
$$

در داده‌های Sentinel-2:

- **Green (سبز)** = باند **B3**
- **NIR (مادون قرمز نزدیک)** = باند **B8**

این شاخص مبتنی بر این واقعیت است که آب بازتاب بسیار کمی در باند NIR و بازتاب بیشتری در باند سبز دارد.

</div>

---

<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 14px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<div class=rtl-content>

#### ب. تفاوت بین NDWI و MNDWI

شاخص MNDWI (Modified NDWI) نسخه‌ی بهبود یافته‌ای از NDWI است که به‌ویژه در مناطق شهری عملکرد بهتری دارد.

فرمول MNDWI:

$$
MNDWI = \frac{Green - SWIR}{Green + SWIR}
$$

در Sentinel-2:

- **Green** = باند B3
- **SWIR (مادون قرمز کوتاه)** = باند B11 یا B12

🔹 تفاوت کلیدی:

- NDWI برای نواحی طبیعی و پوشش‌های گیاهی مناسب‌تر است.
- MNDWI در محیط‌های شهری یا خشک عملکرد دقیق‌تری دارد.

</div>

---

<style>
div.rtl-content {
    font-family: "Tahoma", "Vazirmatn", "IranSans", sans-serif;
    font-size: 14px;
    line-height: 1.8;
    direction: rtl;
    text-align: right;
}
</style>

<div class=rtl-content>

### ج. نقش باند NIR در تشخیص نواحی آبی

در باند NIR، سطح گیاه و خاک بازتاب زیادی دارند، در حالی که **آب بازتاب بسیار کمی دارد** یا حتی کاملاً جذب می‌شود.

به همین دلیل، مقدار پایین NIR می‌تواند نشان‌دهنده وجود آب باشد. این خاصیت باعث می‌شود شاخص‌هایی مانند NDWI و NDVI مؤثر باشند.

---

#### د. محاسبه شاخص‌ها با مقادیر پیکسل داده‌شده
</div>

In [1]:
B3 = 1   # Green
B4 = 9   # Red
B8 = 9   # NIR
B11 = 12 # SWIR

NDVI = (B8 - B4) / (B8 + B4)
NDWI = (B3 - B8) / (B3 + B8)
MNDWI = (B3 - B11) / (B3 + B11)

NDVI, NDWI, MNDWI

(0.0, -0.8, -0.8461538461538461)

<div dir="rtl">

### نتایج:

- **NDVI = 0** → بدون پوشش گیاهی
- **NDWI = -0.8** → نبود آب سطحی
- **MNDWI ≈ -0.846** → ناحیه خشک یا شهری

### ✅ نتیجه‌گیری:

این پیکسل مربوط به منطقه‌ای **خشک، فاقد پوشش گیاهی و منابع آبی سطحی** است.

</div>